src/main/java/mediathek/javafx/filterpanel/JFXSearchPanel.java

```diff
@@ -16,7 +16,7 @@ public JFXSearchPanel() {
 
         setPrefWidth(350d);
         setMinWidth(350d);
-        setMaxWidth(350d);
+        setMaxWidth(500d);
```

Comment:
This change doesn´t have any benefit as it won´t make the field bigger. It will be using the PREF size not the MAX size.

Reply:
Ok I revert it.